wallet/builder.go

```diff
@@ -127,8 +125,24 @@ func NewMultiWallet(cfg *WalletConfig) (multiwallet.MultiWallet, error) {
 					coin.ClientAPIs = []string{cfg.ConfigFile.LTC.APITestnet}
 				}
 			}
-			coinCfgs = append(coinCfgs, coin)
+		case wallet.Ethereum:
+			walletDB := CreateWalletDB(cfg.DB, coin.CoinType)
+			coin.DB = walletDB
+			if cfg.ConfigFile.ETH != nil {
+				coin.FeeAPI = cfg.ConfigFile.ETH.API
+				coin.LowFee = uint64(cfg.ConfigFile.ETH.LowFeeDefault)
+				coin.MediumFee = uint64(cfg.ConfigFile.ETH.MediumFeeDefault)
+				coin.HighFee = uint64(cfg.ConfigFile.ETH.HighFeeDefault)
+				coin.MaxFee = uint64(cfg.ConfigFile.ETH.MaxFee)
+				if !testnet {
+					coin.ClientAPIs = []string{cfg.ConfigFile.ETH.API}
+				} else {
+					coin.ClientAPIs = []string{cfg.ConfigFile.ETH.APITestnet}
+				}
+				coin.Options = schema.EthereumDefaultOptions()
+			}
 		}
+		coinCfgs = append(coinCfgs, coin)
```

Comment:
@amangale This looks like it was missing in both the LTC and ETH cases. Was this intentional?